pkg/modules/auth/openid/openid.go

```diff
@@ -68,6 +71,7 @@ type claims struct {
 	PreferredUsername string                   `json:"preferred_username"`
 	Nickname          string                   `json:"nickname"`
 	VikunjaGroups     []map[string]interface{} `json:"vikunja_groups"`
+	Picture           string                   `json:"picture"`
```

Comment:
Is there a standard for what this is called? If not, this attribute name should be configurable.

Reply:
https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims

`picture` is contained in OIDC standard, no need to be configurable